R-packages/evalcast/DESCRIPTION

```diff
@@ -94,4 +94,4 @@ Suggests:
 VignetteBuilder:
     utils
 Depends:
-    R (>= 3.5.0)
+    R (>= 4.1.0)
```

Comment:
for a hot second I was extremely confused, because the CI changes from this commit block are for covidcast, and this version dependency change is for evalcast

but i've confirmed that the only R version in the evalcast ci is 4.1, so this change should work fine with ci